README.md

```diff
@@ -1,28 +1,13 @@
-# Nexmo Client Library For Go
+# Nexmo Server SDK For Go
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/nexmo-community/nexmo-go)](https://goreportcard.com/report/github.com/nexmo-community/nexmo-go)
 [![Build Status](https://travis-ci.org/nexmo-community/nexmo-go.svg?branch=master)](https://travis-ci.org/nexmo-community/nexmo-go)
 [![Coverage](https://codecov.io/gh/nexmo-community/nexmo-go/branch/master/graph/badge.svg)](https://codecov.io/gh/nexmo-community/nexmo-go)
 [![GoDoc](https://godoc.org/github.com/nexmo-community/nexmo-go?status.svg)](https://godoc.org/github.com/nexmo-community/nexmo-go) 
 
-This library is moving towards a full client implementation of the
-[Nexmo](https://www.nexmo.com/) APIs. The library is
-supported by Nexmo.
+This is the definitely-unofficial Golang library for [Nexmo](https://nexmo.com). It has support for most of our APIs, but is still under active development. Issues, pull requests and other input is very welcome.
```

Comment:
While I _like_ the phrase "definitely-unofficial" it's not really true. The plan is for this library to be fully supported by Nexmo (in fact I think that's in my goals for this year). So it's official, but community-supported right now? Or official but not fully supported by Nexmo.